### PR TITLE
optimize drift check with argocd, replace patch body with github file

### LINKED ISSUE #22: replace patch body with github yaml file for drift-check-with-argocd
## Enhancement

in page drift-check-with-argocd, those two critical patch body were hard code in text, we'd better use online file to trace changes

### PR DESCRIPTION
**Pull request by howieyuen**
### What problem does this PR solve?

Issue Number: close #22 

Problem Summary:

### CODE REVIEW

File: docs/user_docs/guides/argocd/drift-check-with-argocd.md [hunk 1]
```diff
@@ -4,16 +4,10 @@
...
-
-```yaml
-data:
-  configManagementPlugins: |
-    - name: kusion
-      generate:
-        command: ["sh", "-c"]
-        args: ["kusion compile -Y kcl.yaml -Y ci-test/settings.yaml"]
-      lockRepo: true
+1. 下载 patch 文件
```
Comment (chai2010) [inline docs/user_docs/guides/argocd/drift-check-with-argocd.md:7]: add link to **patch**

File: docs/user_docs/guides/argocd/drift-check-with-argocd.md [hunk 2]
```diff
@@ -27,38 +21,10 @@ kubectl -n argocd patch cm/argocd-cm -p "$(cat patch-argocd-cm.yaml)"
...
-        env:
-        - name: PATH
-          value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/kusion/bin:/kusion/kclvm/bin 
-        volumeMounts:
-        - mountPath: /kusion
-          name: custom-tools
-          subPath: kusion
-        securityContext:
-          readOnlyRootFilesystem: false
+1. 下载 patch 文件
```
Comment (chai2010) [inline docs/user_docs/guides/argocd/drift-check-with-argocd.md:24]: add link to **patch**

**Comment by howieyuen** [comment]
> I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

**Comment by howieyuen** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by chai2010 — APPROVED** [review]
LGTM

**Pull request merged**